drivers/periph_common/Kconfig.timer

```
@@ -30,3 +30,6 @@ config MOD_PERIPH_INIT_TIMER_PERIODIC
     default y if MOD_PERIPH_INIT
 
 endif # MOD_PERIPH_TIMER
+
+# Include CPU specific configurations
+orsource "$(RIOTCPU)/$(CPU)/periph/Kconfig.timer"
```

This is an absolute path:
```suggestion
osource "$(RIOTCPU)/$(CPU)/periph/Kconfig.timer"
```